apps/api/src/hyp3_api/api-spec/openapi-spec.yml.j2

```diff
@@ -315,6 +317,12 @@ components:
       type: string
       example: I want to process data.
 
+    access_code:
+      description: Grants automatic user approval while the code remains active.
+      type: string
+      format: uuid
```

Comment:
will need to change to string

Reply:
FYI @forrestfwilliams, we had discussed setting a minimum length for `access_code`, but as far as I can tell there's no way to enforce a minimum length for the DynamoDB field, so it will be up to HyP3 operators to self-enforce a minimum length, which means there's no point in setting a minimum length in the API spec (as this would just restrict what codes users can submit).

Reply:
Yep, that's fine.